fine-uploader/index.d.ts

```diff
@@ -0,0 +1,337 @@
+// Type definitions for FineUploader for 5.11.10
```

Comment:
Lint error is here: `1:25 dt-header Error parsing header. Expected: foo MAJOR.MINOR (patch version not allowed)`
This line should be just `// Type definitions for FineUploader for 5.11`.
Then remove the last line (`// TypeScript Version: 2.1`).